packages/docs/src/utilities/utils.ts

```diff
@@ -13,3 +13,18 @@ export const readComponent = async (fileName: string): Promise<any> => {
     /* webpackMode: "lazy-once" */
     `../examples/${fileName}.vue`)
 }
+
+/**
+* Find `$t` inside a code and replace it with translation by regex.
+* origin: `... $t('to replace') ...`
+* output: `... translated ...`
+*/
+export function applyTranslations (this: any, code: string) {
+  const replaces = code.match(/(?:\$t)\(.*?\)/) || []
```

Comment:
here we need to add a flag `g`:
`const replaces = code.match(/(?:\$t)\(.*?\)/g) || []`